View_Assist_custom_sentences/Play_Music_with_Music_Assistant/blueprint-playmusicwithmusicassistant.yaml

```diff
@@ -24,153 +24,262 @@ blueprint:
-    command:
+    id: track
+    trigger: conversation
+  - command:
       - queue {song} by {artist}
       - cue {song} by {artist}
-    id: cuesong
-condition: []
-action:
+    id: cuetrack
+    trigger: conversation
+  - trigger: conversation
+    command:
+      - turn up the volume
+      - turn up the music
+      - turn up the music volume
+      - increase the music volume
+    id: volume_up
+  - trigger: conversation
+    command:
+      - turn down the volume
+      - turn down the music
+      - turn down the music volume
+      - decrease the music volume
+    id: volume_down
+  - trigger: conversation
+    command:
+      - pause the music
+      - resume the music
+    id: pause
+  - trigger: conversation
+    command:
+      - next song
+      - skip this song
+      - skip to next song
+    id: next
+conditions: []
+actions:
+  - variables:
+      target_musicplayer_device: media_player.picoreplayer
   - choose:
       - conditions:
           - condition: trigger
             id:
-              - playartist
+              - artist
         sequence:
-          - service: python_script.set_state
-            data:
-              entity_id: "{{ target_satellite_device }}"
-              mode: music
-              title: ""
-          - service: browser_mod.navigate
-            metadata: {}
-            data:
-              path: "{{ view_path }}"
-            target:
-              device_id: "{{ target_display_device }}"        
-          - service: music_assistant.search
-            data:
+          - data:
               limit: 1
               config_entry_id: "{{ ma_instance }}"
               media_type:
-                - artist
+                - "{{trigger.id}}"
               name: "{{ trigger.slots.artist  }}"
             response_variable: found_artist
-          - service: music_assistant.play_media
-            target:
-              entity_id: "{{ target_musicplayer_device }}"
-            data:
-              media_type: artist
-              enqueue: replace
-              media_id: "{{ found_artist.artists.0.name }}"
-          - set_conversation_response: ""
+            action: music_assistant.search
+          - alias: Did we find an artist?
+            if:
+              - condition: template
+                value_template: "{{found_artist['artists'] | count>0}}"
+                alias: We found an artist!
+            then:
+              - data:
+                  entity_id: "{{ target_satellite_device }}"
+                  mode: music
+                  title: ""
+                action: python_script.set_state
+              - metadata: {}
+                data:
+                  path: "{{ view_path }}"
+                target:
+                  device_id: "{{ target_display_device }}"
+                action: browser_mod.navigate
+              - target:
+                  entity_id: "{{ target_musicplayer_device }}"
+                data:
+                  media_type: "{{trigger.id}}"
```

Comment:
same question as above.  Does this work?

Reply:
Yes.